docs/source/development/index.rst

```diff
@@ -4,25 +4,21 @@
 Quick start
 -----------
 
-Get the code, work on a new branch::
+To obtain a copy of the code and work on a new branch::
 
     git clone https://github.com/NASA-PDS/pds-doi-service.git
-    git checkout -b "#<issue number>"
+    git checkout -b "<issue number>_<issue name>"
 
-Create a virtual environment in ``venv`` using python 3.7 or later::
+Create a virtual environment in ``venv`` using Python 3.9 or later::
 
     python3 -m venv venv
 
-Install dependencies::
+Install the package and its dependencies for development into the virtual environment::
 
-    venv/bin/pip install --requirement requirements.txt
+    pip install --editable '.[dev]'
 
-and make it ready for development::
-
-    venv/bin/pip install --editable .
-
-At this point, we will have the command line tools available in ``venv/bin``
-as explained in usage.
+At this point, the command line tools are now available in ``venv/bin``
+as explained in `usage`_.
 
 .. note::
```

Comment:
*Actually*, `tox` won't even work unless you _activate_ the virtual environment. It seems the Python community has back-pedaled on this concept. The tides of battle shift again, but the war is not yet won.